src/lando/api/views.py

```diff
@@ -237,4 +239,25 @@ class Form(forms.Form):
         job.status = JobStatus.SUBMITTED
         job.save()
 
-        return JsonResponse({"id": job.id}, status=201)
+
+class PullRequestChecksAPIView(APIView):
+    def get(self, request: WSGIRequest, repo_name: str, number: int) -> JsonResponse:
+        target_repo = Repo.objects.get(name=repo_name)
+        client = GitHubAPIClient(target_repo)
+        pull_request = client.build_pull_request(number)
+
+        patch_helper = PullRequestPatchHelper(client, pull_request)
+
+        landing_checks = LandingChecks(f"{pull_request.user_login}@github-pr")
+        checks = [
+            chk.__name__
```

Comment:
Can we use something other than `__name__` here? A more explicit field (`chk.name` perhaps?)

This may also lead to less conflicts when #552 is merged.